README.md

```
@@ -1,6 +1,6 @@
 # celery-flow 🌊
 
-**A lightweight Celery task flow visualizer**
+**Zero-infrastructure Celery task flow visualizer**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)